gdcdictionary/schemas/germline_mutation_calling_workflow.yaml

```diff
@@ -27,12 +27,19 @@ links:
     target_type: aligned_reads
     multiplicity: many_to_many
     required: true
+  - name: germline_variation_catalogs
+    backref: germline_mutation_calling_workflows
+    label: performed_on
+    target_type: germline_variation_catalog
+    multiplicity: many_to_many
+    required: true
 
+#should be oneOf aligned_reads_files or germline_variation_catalogs, but not supported
```

Comment:
Wait, why couldn't you do an exclusive subgroup? I understand the required props wouldn't work but I'm pretty sure the `uc-cdis/gdcapi` has the same functionality as `NCI-GDC`